frontend/src/_includes/partials/filter-section.html

```diff
@@ -1,11 +1,11 @@
-<div x-data="{ expanded: false }" class="mb-2">
+<div x-data="{ expanded: false }" class="mb-2 rounded-md border border-gray-200">
   <div
     @click="expanded = !expanded"
-    :class="{ 'text-gray-800 bg-gray-100': !expanded }"
-    class="mb-2 flex w-full cursor-pointer rounded-md border border-gray-200 p-2 py-2 pb-3 pt-3 text-left align-baseline text-sm font-medium uppercase text-gray-500 hover:bg-gray-300"
+    :class="!expanded ? 'text-gray-800 bg-neutral-100' : ''"
+    class="flex w-full cursor-pointer rounded-md border border-gray-200 px-2 py-[10px] text-left align-baseline text-sm font-medium uppercase text-gray-500 hover:bg-neutral-300"
```

Comment:
I'm not sure using pixels here is justified. In general, we try to avoid it. Try to find some replacement.